source/submodules.rst

```diff
@@ -30,6 +30,11 @@ the list on the left.
 |Remove        | Remove the submodule from the repository                                                                        |
 +--------------+-----------------------------------------------------------------------------------------------------------------+
 
+Change submodule path
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Delete the existing submodule and add it again in the new location after changing the submodule location.
+
+
```

Comment:
Suggest not adding a header (the ^^^^^ length should match the text exactly if kept) and to reword a little.
```suggestion
To change a submodule path, delete the existing submodule, move the filesystem directory and add it again in the new location.

```